tests/View/Blade/BladePhpStatementsTest.php

```diff
@@ -22,6 +22,13 @@ public function testPhpStatementsWithoutExpressionAreIgnored()
         $this->assertEquals($expected, $this->compiler->compileString($string));
     }
 
+    public function testReturnStatements()
+    {
+        $string = '@return';
+        $expected = '<?php return; ?>';
+        $this->assertEquals($expected, $this->compiler->compileString($string));
```

Comment:
```suggestion

        $this->assertEquals($expected, $this->compiler->compileString($string));
```

For assertion readability, below codes did that too.

Reply:
Has the same format as existing methods.